conf/application.conf

```diff
@@ -148,6 +156,7 @@ microservice {
 
 features {
   allowListEnabled: false
+  chrisStub: false
```

Comment:
This isn't needed

Reply:
it says in the story to ensure that the route is defaulted to Chris

Reply:
The feature switch is off by default so it will be defaulted to use Chris. You don't need to add the config locally, only in the envs if you want to change the defaults.

Reply:
I am aware how the environment configs work,  I would of personally thought it would it would be clearer to read, ill make the change if you insist however we have taken this approach of explicitly defining the state of the feature switch in the front end services so we may want to remove them there too

Reply:
I'm not sure how much value there is in having a default local config as you'll likely enable the features you need as and when you need to during development and the ATs will change them as needed during the course of the tests, however, as this seems to be the standard within the team, I'll align the other frontend to this rather than removing it from two frontends.